picard/ui/options/interface.py

```diff
@@ -169,6 +171,12 @@ def __init__(self, parent=None):
         super().__init__(parent)
         self.ui = Ui_InterfaceOptionsPage()
         self.ui.setupUi(self)
+
+        self.ui.ui_theme.clear()
+        for theme in AVAILABLE_UI_THEMES:
+            self.ui.ui_theme.addItem(_(theme_enum_to_label(theme)), theme)
+        self.ui.ui_theme.setCurrentIndex(0)
```

Comment:
It shouldn't change much, but better select the actual default theme, rather than the first entry.

```suggestion
        self.ui.ui_theme.setCurrentIndex(self.ui.ui_theme.findData(UiTheme.DEFAULT))
```